log4j.properties

```diff
@@ -0,0 +1,8 @@
+# variables are substituted from filters-[production|development].properties
```

Comment:
log4j framework won't find this config at this path. Please name this file `log4j2.properties` (log4j v2) and move it to src/test/resources folder.
Content example you can find at https://github.com/folio-org/mod-invoice/blob/master/src/test/resources/log4j2.properties